### PR TITLE
Add example collapsible section that is open by default

### LINKED ISSUE #34836: Improve collapsible section content by adding example of one that is open by default
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/.github/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections

### What part(s) of the article would you like to see updated?

Add a section detailing how to create a collapsible section that is open by default.

I've created a PR #34837.

### Additional information

This is to add content.

### PR DESCRIPTION
**Pull request by mjbear**
### Why:

Closes: #34836

Also I searched and there were no existing issues or PRs.

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Added content that provides an example of how to create a collapsible section that is open by default.

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Review by mjbear — COMMENTED** [review]
Fixed blank lines around heading and code fence

**Comment by Blackboy007are** [comment]
mjbear:collapsible_section_open_by_default

**Comment by ratchanonsuttawas** [comment]
> 
> 
> ### Why:
> 
> Closes: #34836
> 
> 
> Also I searched and there were no existing issues or PRs.
> 
> ### What's being changed (if available, include any code snippets, screenshots, or gifs):
> 
> 
> 
> Added content that provides an example of how to create a collapsible section that is open by default.
> 
> ### Check off the following:
> 
> - [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).
> 
>   - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
> - [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).
>

**Comment by ratchanonsuttawas** [comment]
> mjbear:collapsible_section_open_by_default

**Review by ratchanonsuttawas — COMMENTED** [review]
+20 -0

**Review by ratchanonsuttawas — COMMENTED** [review]
+20 -0

**Review by ratchanonsuttawas — COMMENTED** [review]
+20 -0

**Comment by Blackboy007are** [comment]
#34837

၂၀၂၄၊ အောက် ၆၊ တနင်္ဂနွေ ၀၀:၅၉ တွင် ratchanonsuttawas <
***@***.***> က ရေးခဲ့သည်−

File: content/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections.md [hunk 1]
```diff
@@ -43,6 +43,26 @@ After a reader clicks {% octicon "triangle-right" aria-label="The right triangle
...
+You can add text within a collapsed section. 
+
+You can add an image or a code block, too.
+
+```python
+   print("Hello World")
+```
+
+</details>
+````
```
Comment (subatoi) [inline content/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections.md:64]:
```suggestion
Optionally, to make the section display as open by default, add the `open` attribute to the `<details>` tag:

```html
<details open>
```

Suggestions don't play nicely with other triple backticks, so you'll have to add three to close the `html` environment.

**Review by subatoi — DISMISSED** [review]
Hi, thanks for your interest in the Docs—I'd gently recommend opening an issue and giving us a chance to examine it and validate it in future before raising a PR, so everybody has a better experience 😄. I do understand that since it's not subject to an SLA, it can involve a bit of wait, though

Your suggested contnet is actually covered in https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/quickstart-for-writing-on-github#adding-a-collapsed-section ... clearly we need to organize the "getting started" information better, but we are aware of that.

I don't have any objection to adding a bit of content here, but it'd be better to keep it simple, so if you're happy with the suggestion I've made, we can go ahead and get this merged 👍

File: content/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections.md (hunk 1 above)
Comment (mjbear) [reply content/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections.md:64]: Hello again @nguyenalex836 :wave: 

Hello @subatoi 
Oh dang, somehow I didn't manage to find that Quickstart page. :disappointed:
Apologies for the lack of time on the issue ticket.
I agree it is best to keep it simple and I agree with your suggestion.
Thank you!

There are lots of docs. Corral them and hyperlink 'em, right? :sweat_smile:
I'm wiser to the next time and wish you all the best in arranging/revising the GH docs. :saluting_face:

**Review by nguyenalex836 — APPROVED** [review]

**Comment by nguyenalex836** [comment]
@mjbear Thank you for applying @subatoi's suggestion! I'll get this merged 💛

**Pull request merged**